acceptance-setup/__tests__/__snapshots__/acceptance.js.snap

```diff
@@ -65,21 +65,18 @@ exports[`server-side-side rendering 1`] = `
             <title>A page</title>
           </head>
           <body>
-            <oc-component href=\\"http://localhost:3000/base-component-es6?name=SuperMario\\" data-name=\\"base-component-es6\\" data-rendered=\\"true\\" data-version=\\"6.6.6\\"><div id=\\"base-component-es6\\" class=\\"awesome__2cThMnJY\\">
```

Comment:
somehow this was broken. When container=false the container should be hidden. Now that is properly working and the snapshot has been updated